instrumentation/dubbo-rpc/pom.xml

```diff
@@ -32,7 +32,7 @@
     <main.java.version>1.6</main.java.version>
     <main.signature.artifact>java16</main.signature.artifact>
     <!-- Use brave-instrumentation-dubbo for Apache Dubbo 2.7+, not this module. -->
-    <dubbo.version>2.6.7</dubbo.version>
+    <dubbo.version>2.6.8</dubbo.version>
```

Comment:
I hadn't realized Dubbo are still releasing the Alibaba library.. I bumped just to make sure they are still not adding error codes.